MaxText/train.py

```diff
@@ -260,7 +266,7 @@ def train_loop(config, state=None):
                        data_pspec,
                        None),
     out_shardings=(state_mesh_annotations, None, None),
-    static_argnums=(0,1,),
+    static_argnums=(0,1,5,),
```

Comment:
why we shouldn't have new args floating around.

Reply:
Removed this